python/ray/rllib/evaluation/worker_set.py

```diff
@@ -31,7 +32,8 @@ def __init__(self,
                  trainer_config=None,
                  num_workers=0,
                  logdir=None,
-                 _setup=True):
+                 _setup=True,
+                 seed=None):
```

Comment:
Can we make sure any underscore args always go at the end of the list, here and elsewhere?